projects/igniteui-angular/src/lib/grids/grid-base.directive.ts

```diff
@@ -3539,17 +3540,29 @@ export abstract class IgxGridBaseDirective extends DisplayDensityBase implements
 
         this.resizeNotify.pipe(
             filter(() => !this._init),
-            throttleTime(40, animationFrameScheduler, { leading: false, trailing: true }),
+            throttleTime(40, animationFrameScheduler, { leading: true, trailing: true }),
             destructor
         )
-            .subscribe(() => {
-                this.zone.run(() => {
-                    // do not trigger reflow if element is detached.
-                    if (this.document.contains(this.nativeElement)) {
-                        this.notifyChanges(true);
+        .subscribe(() => {
+            this.zone.run(() => {
+                // do not trigger reflow if element is detached.
+                if (this.document.contains(this.nativeElement)) {
```

Comment:
Technically, `this.nativeElement.isConnected` should give you the same thing, right?